src/cf-forms/src/atoms/text-input.less

```diff
@@ -1,7 +1,7 @@
 .a-text-input {
     .u-webfont-regular();
 
-    // Browser Reset
```

Comment:
Fixed in https://github.com/cfpb/capital-framework/pull/499/commits/2d399cde7207aca5584be365852f4c815b685303